### ISSUE TITLE
Notification commands provided by `apputils` seem to not be available

### ISSUE THREAD

**Issue by ibdafna** [jupyterlab/jupyterlab#13495]
## Description

Commands provided by the `apputils` package, used to create and update notifications seem to not be available in JupyterLab 3.5. Importing the `Notifications` module from `'@jupyterlab/apputils'` also resolves to undefined. The extension guide has an example of this notification command: https://jupyterlab.readthedocs.io/en/stable/extension/ui_helpers.html#using-commands

## Reproduce

Test out in the JupyterLab Plugin Playground with the following snippet:
```typescript
import {
  JupyterFrontEnd,
  JupyterFrontEndPlugin
} from '@jupyterlab/application';

const plugin: JupyterFrontEndPlugin<void> = {
  id: 'hello-world:plugin',
  autoStart: true,
  activate: async (app: JupyterFrontEnd) => {
    app.commands.execute('apputils:notify', {
      message: "We're working on this!";
      type: 'in-progress';
      options: {
        autoClose: false;
      };
    });

    console.log(app.commands)
  },
};

export default plugin;
```

<img width="479" alt="image" src="https://user-images.githubusercontent.com/24281433/204077295-ddff2149-691f-40ca-99d4-35646575e487.png">

Apputils commands available:
<img width="504" alt="image" src="https://user-images.githubusercontent.com/24281433/204077328-c2763350-d060-48f5-a306-69a2fd9569ad.png">

## Expected behavior

I believe a notification should render.

## Context

- Operating System and version: Ubuntu 20.04
- Browser and version: MS Edge latest
- JupyterLab version: 3.5.0

**Comment by jtpio** [comment]
Thanks @ibdafna.

According to https://github.com/jupyterlab/jupyterlab/pull/12959, it looks like the PR was only backported to `3.6.x` but not `3.5` which can explain why it's not available.

**Comment by fcollonval** [comment]
It seems that the 3.6.0 pre releases are set to be the stable channel on read the docs. Hence the misleading documentation. Sorry for that @ibdafna

**Comment by jweill-aws** [comment]
Closing because this is a versioning issue; fixed in 3.6.x. Thanks!
